### PR TITLE
[Merged by Bors] - Add DirectoryUserUpdated webhook event

### PR DESCRIPTION
**Pull request by sheldonvaughn**

### CODE REVIEW

File: src/webhooks/types/events/directory_user_updated.rs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+/// Type for a [`DirectoryUser`] with previous attributes object returned in a [`dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
+pub struct DirectoryUserWithPreviousAttributes {
+    #[serde(flatten)]
+    /// [`DirectoryUser`] that will be included in the [`DirectoryUserWithPreviousAttributes`]
+    pub directory_user: DirectoryUser,
+
+    /// Type for previous attributes that were updated in a [`dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
+    pub previous_attributes: HashMap<String, Value>,
+}
+/// [WorkOS Docs: `dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
```
Comment (maxdeviant) [inline src/webhooks/types/events/directory_user_updated.rs:19]: Could we add a blank line here?

```suggestion

/// [WorkOS Docs: `dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
```

File: src/webhooks/types/events/directory_user_updated.rs [hunk 2]
```diff
@@ -0,0 +1,125 @@
+use std::collections::HashMap;
+
+use serde::Deserialize;
+use serde_json::Value;
+
+use crate::directory_sync::DirectoryUser;
+
+#[derive(Debug, PartialEq, Eq, Deserialize)]
```
Comment (maxdeviant) [inline src/webhooks/types/events/directory_user_updated.rs]: Could we move the derives down below the doc comment?

File: src/webhooks/types/events/directory_user_updated.rs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+use serde::Deserialize;
+use serde_json::Value;
+
+use crate::directory_sync::DirectoryUser;
+
+/// Type for a [`DirectoryUser`] with previous attributes object returned in a [`dsync.user.updated` Webhook](https://workos.com/docs/reference/webhooks/directory-user#webhooks-dsync.user.updated)
+#[derive(Debug, PartialEq, Eq, Deserialize)]
+pub struct DirectoryUserWithPreviousAttributes {
+    #[serde(flatten)]
+    /// [`DirectoryUser`] that will be included in the [`DirectoryUserWithPreviousAttributes`]
```
Comment (maxdeviant) [inline src/webhooks/types/events/directory_user_updated.rs:11]: Could we swap the order of this comment and the attributes above it?

**Review by maxdeviant — APPROVED** [review]

**Comment by sheldonvaughn** [comment]
bors merge

**Pull request closed without merging**